cluster/images/hyperkube/static-pods/kube-proxy.json

```diff
@@ -0,0 +1,27 @@
+{
+  "apiVersion": "v1",
+  "kind": "Pod",
+  "metadata": {
+    "name": "k8s-proxy",
+    "namespace": "kube-system"
+  },
+  "spec": {
+    "hostNetwork": true,
+    "containers": [
+      {
+        "name": "kube-proxy",
+        "image": "gcr.io/google_containers/hyperkube-ARCH:VERSION",
+        "command": [
+                "/hyperkube",
+                "proxy",
+                "--master=http://127.0.0.1:8080",
```

Comment:
See kubernetes/kube-deploy#164
When that PR is in, it will be fixed.

But we should investigate along with @mikedanese if/when we're moving kube-proxy to a DaemonSet for_all_ (or most) deployments.